filebeat/docs/inputs/input-container.asciidoc

```diff
@@ -0,0 +1,64 @@
+:type: container
+
+[id="{beatname_lc}-input-{type}"]
+=== Container input
```

Comment:
@dedemorton please have a look at the docs changes. FF for 7.2 is pretty close, maybe we can cleanup docs later?

Reply:
I'm ok with merging now and opening folloup PRs for documentation

Reply:
I was on vacation when this request came through and didn't have a chance to review the doc changes. I've created https://github.com/elastic/beats/issues/12244 to get the content edited.